client/src/components/Monument/Monument.js

```diff
@@ -9,27 +11,53 @@ export default class Monument extends React.Component {
     }
 
     async componentDidMount() {
-        const { match: { params } } = this.props;
-
-        const response = await fetch(`/api/monument/${params.monumentId}`);
+        const { match: { params: { monumentId, slug } } } = this.props;
+        const response = await fetch(`/api/monument/${monumentId}`);
         const monument = await response.json();
+        console.log(monument);
```

Comment:
Debug comment?

Reply:
Intentionally left it in, it will be nice to have easy access to that info going forward. I could add a todo to get rid of it eventually